### ISSUE TITLE
Additional theme argument gone?

### ISSUE THREAD

**Issue by Lokua** [paypal/glamorous#293]
- `glamorous` version: 2.20.40
- `glamor` version: 4.2.1
- `react` version: 15.6.1

Relevant code.

```javascript
// assuming higher up <ThemeProvider theme={{ color: 'red' }}>..

glamorous.div((props, theme) => {
  theme.color === 'red' // => false
  theme // => { __glamorous__: {...}, __proto__: Object }
  return {/**/}
})
```

What you did:

Upgrade to glamorous 4.2.1 from 3.19.0

What happened:

The second theme argument to styles function is no longer a theme. I can only access theme via props, which works, but requires I refactor most of my components made with v3.

Reproduction:

https://codesandbox.io/s/m8jw3x50y

Problem description:

According to https://github.com/paypal/glamorous/issues/198, theme was added to props _in addition_ to having theme as second argument. Is this supposed to longer be the case or is this a bug?

Suggested solution:

Ensure theme can still be used as second argument in styles function

Additional remarks:
This is currently how majority of my glamorous components are defined:

```js
import g from 'glamorous'

const Container = g.div((p, t) => ({ /* ... */ }))
```

Normally I prefer using complete names for imports and arguments as it's good practice, but for something like this where I'm writing 100s of css components, it's really nice to be able to use `p` and `t`. Not looking forward to having to destructure every time in the future if that indeed is the only way, but whatever, I can adjust.

**Comment by tdeschryver** [comment]
Hello, the `theme` argument was indeed removed in #253 (you can take a look at breaking changes for more info).
Thankfully @kentcdodds created a codemod which you can find [here](https://github.com/paypal/glamorous/tree/master/other/codemods#theme-move) that you can run on your code which will transform the code for you!

TLDR:
Before: `glamorous.div((props, theme) => ({fontSize: theme.fontSize}))`
After: `glamorous.div((props) => ({fontSize: props.theme.fontSize}))`

**Comment by Lokua** [comment]
Ah. Should've looked harder. Thanks.
